api/topo/topo.proto

```diff
@@ -118,6 +119,7 @@ message Object {
 message Entity {
   // user-defined entity kind
   string kind_id = 1 [(gogoproto.customname) = "KindID", (gogoproto.casttype) = "ID"];
+  repeated topo.device.ProtocolState protocols = 2;
```

Comment:
Maybe you should be moving this across in to the new hierarchy. I was wondering if we were going to bring this over in to the new hierarchy at all, but I think it is worthwhile. `onos-ric` also needs to update this protocol state like it is done here for `onos-config`

Reply:
https://github.com/onosproject/onos-topo/issues/168